Source/Core/Common/Network.cpp

```diff
@@ -18,6 +18,15 @@
 
 #include <fmt/format.h>
 
+#ifdef __linux__     // features.h is only available on Linux
+#define _GNU_SOURCE  // We have to define this to use macros in features.h
+#include <features.h>
+#ifndef __USE_GNU
```

Comment:
Won't this have side-effects on other Linux distributions such as forcing the use of the GNU version? Isn't there any other way than relying on the GNU features to check if the musl libc implementation is used?

Reply:
> Won't this have side-effects on other Linux distributions such as forcing the use of the GNU version?

I'm not sure what you mean here. Also, the Linux CI on this repository (which seems to use something Debian-based) built fine, so I'm pretty confident it'll work on any other distribution.

> Isn't there any other way than relying on the GNU features to check if the musl libc implementation is used?

Nope. I wasn't able to find any other way.

Reply:
This might be worth adding as a comment in the code:
```c++
// https://stackoverflow.com/a/70211227
```

I was surprised that `__USE_GNU` wasn't defined after `_GNU_SOURCE` being defined.

Reply:
Oh yeah, I got the code from there, I just didn't think it was "nececssary" for such a small contribution/code.